tests/fast-integration/application-connectivity-2-test/lastorder-function.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: serverless.kyma-project.io/v1alpha1
```

Comment:
Copy&paste?
Why can't you reuse commerce-mock fixture?

Reply:
Right! The env is still provided but I think I can use a similar approach to [this one](https://github.com/kyma-project/kyma/pull/11152/files#diff-f05d98841f4a63b71b83a91fb79bc0b79651616cde2938a897a6bdb46b931aa3R80-R85). The URL part of the lambda will be just created in the `fixtures/commerce-mock/index.js` functions depending on the scenario

Reply:
Done